subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/DependencyReportTask.java

```diff
@@ -16,11 +16,13 @@
 package org.gradle.api.tasks.diagnostics;
 
 import org.gradle.api.artifacts.ConfigurationContainer;
+import org.gradle.work.DisableCachingByDefault;
 
 /**
  * Displays the dependency tree for a project. An instance of this type is used when you
  * execute the {@code dependencies} task from the command-line.
  */
+@DisableCachingByDefault(because = "Not made cacheable, yet")
```

Comment:
> Isn't this similar to HtmlDependencyReportTask, doing most of the work in snapshotting?

Yeah, it probably is. Note that both tasks don't declare the dependency graph as input, so both won't do snapshotting anyway.

> Curious: why does DependencyReportTask and HtmlDependencyReportTask not share a common ancestry?

I suppose because they are too different. One is a report mostly used for console output/plain text output of a single project configurable by command line option why the second one is producing html output for multiple projects for all configurations.